Kuzzle/API/Controllers/CollectionController.cs

```diff
@@ -177,7 +177,7 @@ public class CollectionController : BaseController {
         { "body", specifications }
       });
 
-      return (JObject)response.Result[index][collection];
+      return (JObject)response?.Result?[index]?[collection];
```

Comment:
@scottinet 

> (nitpicking) unnecessary: if there is no error, then the {index: {collection: { ... } } objects are always defined.

(Sorry I deleted your review by accident)

In fact there is an error for the UpdateSpecification snippet and I don't know why, `response.Result[index][collection]` throw an NPE